src/zc/buildout/tests/update.txt

```diff
@@ -275,11 +273,11 @@ gives us a sys.exit:
 
 Run the buildout:
 
-    >>> print_(system(buildout, with_exit_code=True), end='')
+    >>> print_(system(buildout, with_exit_code=True), end='') # doctest: +ELLIPSIS -DONT_ACCEPT_BLANKLINE
     Upgraded:
-      zc.buildout version 1.4.4,
+      zc.buildout V.V
     Restarting.
     Generated script '/sample-buildout/bin/buildout'.
     Develop: '/sample-buildout/failrecipe'
-    recipe sys-exits
+    ...ecipe sys-exits
```

Comment:
Why did you change this?  Did you get `Recipe` with a capital R locally?  That would be weird, because the recipe in line 239 just does `sys.exit('recipe sys-exits')`.

I don't really mind the change too much, because I know these doctests can be tricky to get green.  Just wondering.